backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/RD3v2Loader.java

```diff
@@ -32,6 +33,7 @@ public void run() {
       }
       List<Row> rows = getProfilesFromAllModels("/portal", List.of());
       getSchema().migrate(Emx2.fromRowList(rows));
+      MolgenisIO.fromClasspathDirectory("/_ontologies", getSchema(), false);
```

Comment:
I'm not sure if this is the correct approach for the profiles and other data models. I used the other loaders as a guide.

Reply:
yes it is.